### PR TITLE
[DOP-3006] add redirect for realm 

### PR DESCRIPTION
**Pull request by rkhullar-mongodb**
## Related Links
- [DOP-3006](https://jira.mongodb.org/browse/DOP-3006)
- [DOP-2956](https://jira.mongodb.org/browse/DOP-2956)

## Description
Add redirect from `mongodb.com/docs/realm/cloud/*` to `mongodb.com/docs/app-services/*`.

### CODE REVIEW

**Comment by rkhullar-mongodb** [comment]
For now I've manually updated the s3 config for the two buckets with the redirect rules.

**Comment by schmalliso** [comment]
@rkhullar-mongodb Revert the manual changes. They're redirecting pages that should not be included in the redirects.

File: infrastructure/ecs-main/serverless.yml [hunk 1]
```diff
@@ -156,6 +156,19 @@ custom:
     prd: "prd"
     dotcomstg: "preprd"
     dotcomprd: "prd"
+  site:
+    host:
+      dev: "docs-dev.mongodb.com" # NOTE: cannot be empty in redirect
```
Comment (casthewiz) [inline infrastructure/ecs-main/serverless.yml:161]: Is this intended to be a future dev environment?

File: infrastructure/ecs-main/serverless.yml [hunk 2]
```diff
@@ -156,6 +156,19 @@ custom:
...
+      stg: "mongodbcom-cdn.website.staging.corp.mongodb.com"
+      dotcomstg: "mongodbcom-cdn.website.staging.corp.mongodb.com"
+      prd: "www.mongodb.com"
+      dotcomprd: "www.mongodb.com"
+    prefix:
+      dev: "docs-dev"
```
Comment (casthewiz) [inline infrastructure/ecs-main/serverless.yml:167]: I think this should be just `docs` - that way we can have `docs-dev.mongodb.com/docs`

**Review by casthewiz — APPROVED** [review]
I think this is good - we should be able to test the redirects in staging at `https://mongodbcom-cdn.website.staging.corp.mongodb.com/docs-qa/realm/` and its cloud paths 😄

File: infrastructure/ecs-main/serverless.yml (hunk 1 above)
Comment (rkhullar-mongodb) [reply infrastructure/ecs-main/serverless.yml:161]: So this is intentionally not valid url currently. If we setup a future domain for dev, then I think this one would be a good fit. I could have used the actual s3 website url domain here, but that would include our aws account id.

File: infrastructure/ecs-main/serverless.yml (hunk 2 above)
Comment (rkhullar-mongodb) [reply infrastructure/ecs-main/serverless.yml:167]: I'll update this. Would we want to eventually do the same for staging? Move from `{stage-domain}/docs-qa` to `{stage-domain}/docs`?

**Pull request closed without merging**